vercel.json

```diff
@@ -10,5 +10,6 @@
             "src": "/(.*)",
             "dest": "dist/app.js"
         }
-    ]
+    ],
+    "regions": ["cdg1"]
```

Comment:
deployed version keeps timing out; possible fix is to update the region to be closer to our data source.